interface/patient_file/encounter/superbill_custom_full.php

```diff
@@ -283,7 +283,7 @@ function bucks($amount)
 <head>
     <title><?php echo xlt("Codes"); ?></title>
 
-    <?php Header::setupHeader(['jquery-ui', 'jquery-ui-base']); ?>
+    <?php Header::setupHeader([]); ?>
```

Comment:
Remove [] brackets as they aren't needed. Should be just `Header::setupHeader();`.

Reply:
Also this file uses JQUI autocomplete which needs removal

Reply:
I am reading your PR to get the better idea of what needs to be changed \ (•◡•) /.